diff --git a/dist/utils.js b/dist/utils.js
index b54e3f7aa14c25b41a62a2f07118404b87dfa2b8..8ac8c36df40fd5f13f26c9fdc9617605c2e524e4 100644
--- a/dist/utils.js
+++ b/dist/utils.js
@@ -2,7 +2,7 @@ import { generateId, } from "better-auth";
 import { getSessionFromCtx } from "better-auth/api";
 import { setSessionCookie } from "better-auth/cookies";
 import { generateRandomString } from "better-auth/crypto";
-import { createAuthMiddleware, } from "better-auth/plugins";
+import { createAuthMiddleware, } from "better-auth/api";
 import { ERROR_CODES } from "./constants";
 export const resolveInviteOptions = (opts) => ({
     getDate: opts.getDate ?? (() => new Date()),
